webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -22,6 +23,21 @@ export const formatValue = variable => {
     ? variable.currentValue.value
     : variable.defaultValue;
 
+  if (variable.hiddenValue) {
```

Comment:
Is the `hiddenValue` necessary here, or could we rely on `variable.meta.canEdit` instead?

Reply:
Unfortunately, the `hiddenValue` field is needed to decide whether the "fake" variable should be shown instead of the real one. Relying on `canEdit` here would not work, as a permitted user can edit hidden and "not-hidden" variables.

Reply:
On second thought, this block isn't necessary. If the user is authorized to view hidden values, we can simply display them as is. If they aren’t authorized, the value should already be hidden by the backend before it's sent.